cyences_app_for_splunk/README/alert_actions.conf.spec

```diff
@@ -6,10 +6,12 @@ param.filter_macro_value = <str> Filter macro value
 param.email_to = <str> Comma separated email addresses
 param.cyences_severities =  <str> Comma separated severities. ie. critical,high,medium,low,info
 param.exclude_alerts = <str> Comma separated alert names to exclude from digest email
+param.subject_prefix = <str> Text to add as a prefix of email subject
```

Comment:
Because its a generic configuration, we need to put into some other file I think.

We can use not only for email but other places, so lets not store here.